msal/tests/dev apps/XForms/XForms/AcquirePage.xaml.cs

```diff
@@ -60,12 +60,12 @@ protected override void OnAppearing()
 
         private void RefreshUsers()
         {
-            var userIds = App.MsalPublicClient.GetAccountsAsync().Result.
-                Select(x => x.Username).ToList();
+            //var userIds = App.MsalPublicClient.GetAccountsAsync().Result.
```

Comment:
I had to comment this out. the app was blowing up here, which has happened periodically in the past when manual testing. Will look into this more tomorrow. want to see if things pass in app center.